src/Components/ServiceScene/Breakdowns/Patterns/PatternFieldLabelStats.tsx

```diff
@@ -0,0 +1,95 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { LogLabelStatsModel, GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+//Components
+import { PatternFieldLabelStatsRow } from './PatternFieldLabelStatsRow';
+
+const STATS_ROW_LIMIT = 10;
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    logsStats: css`
```

Comment:
Some example values can get really long, pushing the content off the page
<img width="1728" alt="image" src="https://github.com/grafana/explore-logs/assets/109082771/5e009500-610d-4084-8f19-21d21ad40840">

Setting a max height and overflow: auto will allow the modal to scroll when the content doesn't fit. 

e.g.
```
      overflow: auto;
      max-height: 80vh;
```
80vh might be too tall, something we should run by @zizzpudding

Reply:
Done, set to 40vh.

![Scrolla](https://github.com/grafana/explore-logs/assets/1069378/53de7535-7661-4b2f-a3cd-b4113a330c34)